src/components/WebexRemoteMedia/WebexRemoteMedia.scss

```diff
@@ -1,17 +1,26 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media {
   min-height: $WEBEX_MEETING_MIN_HEIGHT;
   min-width: $WEBEX_MEETING_MIN_WIDTH;
-  background: white;
+  background: var(--wxc-video-background);
 
   position: relative;
   display: flex;
   justify-content: center;
   align-items: center;
 
-  .md-alert-banner {
-    position: absolute;
-    top: 0;
-    width: 100%;
+  .media-error {
+    font-family: $brand-font-bold;
+    font-size: 0.875rem;
+    line-height: 1.3125rem;
+    color: var(--wxc-text-color);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
```

Comment:
`flex-direction: column` should come right after `display: flex`, because it specifies "what kind of flex" we're using and gives meaning to `justify-content` and `align-items` (which have different meanings based on flex direction).